api/nodes/serializers.py

```diff
@@ -373,3 +379,29 @@ class Meta:
     @staticmethod
     def get_id(obj):
         return '{}:{}'.format(obj.node._id, obj.provider)
+
+class NodeInstitutionRelationshipSerializer(JSONAPIRelationshipSerializer):
+
+    class Meta:
+        type_ = 'institution'
+
+    def relationship(self, obj):
+        return obj.primary_institution
+
+    def update(self, instance, validated_data):
+        node = instance
+        user = self.context['request'].user
+        data = self.context['request'].data['data']
+        if data:
```

Comment:
If you add a parse_method to JSONAPIRelationshipParser, you could do a check like, if validated_data.get('_id).